pages/dashboard/introduction.md

```diff
@@ -0,0 +1,110 @@
+# Introdução
+
+O [Cypress Dashboard](https://on.cypress.io/dashboard) é um serviço que permite que você tenha acesso a 
+testes gravados - normalmente ao executar os testes do Cypress de seu provedor de integração contínua (CI).
+
+[![Visão Geral do Cypress Dashboard.](https://img.youtube.com/vi/ezp60FUnjGg/0.jpg)](https://youtu.be/ezp60FUnjGg)
+
+>  Real World Example
```

Comment:
Real World Example is not translated.

Reply:
@lucastfa-toptal since it's the name of cypress application. Me and @wlsf82 already discussed about this and we think that's better to keep the same name. Or do you mind that I can change to "Exemplo Real World"?

@wlsf82 let us know your opinion please

Reply:
The name is Real World right? Or Example is also in the name? So maybe `Exemplo do Real World` would be ok.

Reply:
The name is Real World App (RWA), but I'll follow your suggestion :)